.github/workflows/chainkeys-testing-canister.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           pocket-ic-server-version: "7.0.0"
       - name: Build chainkeys-testing-canister Darwin
         run: |
-          cargo build --target wasm32-unknown-unknown --release
+          dfx build --check
```

Comment:
In the Makefile, the build command is still `cargo build --target wasm32-unknown-unknown --release`, but maybe it is better to leave this, as there we don't need the gzip.